src/extension.ts

```diff
@@ -236,8 +235,8 @@ async function activateServerForFolder(context: ExtensionContext, uri: Uri, fold
   // For our intents and purposes, the server should be launched the same way in
   // both debug and run mode.
   const serverOptions: ServerOptions = {
-    run: { command: serverExecutable, transport: TransportKind.stdio, args, options: exeOptions },
-    debug: { command: serverExecutable, transport: TransportKind.stdio, args, options: exeOptions },
+    run: { command: serverExecutable, args, options: exeOptions },
+    debug: { command: serverExecutable, args, options: exeOptions },
```

Comment:
@July541 That's the necessary change right? I ran it locally, and it worked.

Reply:
Yes! everything should be find without `TransportKind.stdio`.